pkg/api/v1/schema.go

```diff
@@ -67,6 +67,7 @@ type Devices struct {
 	Disks      []Disk      `json:"disks,omitempty"`
 	Serials    []Serial    `json:"serials,omitempty"`
 	Consoles   []Console   `json:"consoles,omitempty"`
+	Watchdog   *Watchdog   `json:"watchdog,omitempty"`
```

Comment:
Could you add documentation to the fields?

Reply:
done. I added them to the struct.